src/main/java/org/apache/mesos/specification/Named.java

```diff
@@ -0,0 +1,8 @@
+package org.apache.mesos.specification;
+
+/**
+ * A named object provides a name.
```

Comment:
To illustrate, among interfaces edited in this PR, each of `Phase`, `TaskSpecification`, and `ServiceSpecification` could all also extend `Named` as-is but I don't think we'd want those to all be jumbled into a single collection together.